src/main/java/mate.academy/service/impl/ProductServiceImpl.java

```diff
@@ -2,13 +2,18 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import mate.academy.lib.Component;
+import mate.academy.lib.Inject;
 import mate.academy.model.Product;
 import mate.academy.service.FileReaderService;
 import mate.academy.service.ProductParser;
 import mate.academy.service.ProductService;
 
+@Component
 public class ProductServiceImpl implements ProductService {
+    @Inject
     private ProductParser productParser;
+    @Inject
     private FileReaderService fileReaderService;
```

Comment:
The variable names 'productParser' and 'fileReaderService' are clear and informative, which is good. However, make sure that these fields are actually being injected by your Injector class and that the Injector is checking for the @Component and @Inject annotations properly.